src/toil/test/src/importExportFileTest.py

```diff
@@ -16,62 +16,83 @@
 
 import uuid
 
-import shutil
-
-import os
 from toil.common import Toil
 from toil.job import Job
-from toil.test import ToilTest
-import tempfile
+from toil.leader import FailedJobsException
+from toil.test import ToilTest, make_tests
 
 
 class ImportExportFileTest(ToilTest):
     def setUp(self):
-        self.srcFile = "%s/%s" % (tempfile.mkdtemp(), uuid.uuid4())
-        with open(self.srcFile, 'w') as f:
-            f.write(os.urandom(2**10))
-
-    def tearDown(self):
-        shutil.rmtree(self.srcFile.rsplit('/', 1)[0])
+        super(ImportExportFileTest, self).setUp()
+        self._tempDir = self._createTempDir()
+        self.dstFile = '%s/%s' % (self._tempDir, 'out')
 
-    def testImportExportFile(self):
+    def _importExportFile(self, options, fail):
+        with Toil(options) as toil:
+            if not options.restart:
+                # stage files
+                srcFile = '%s/%s%s' % (self._tempDir, 'in', uuid.uuid4())
+                with open(srcFile, 'w') as f:
+                    f.write('Hello')
+                inputFileID = toil.importFile('file://' + srcFile)
+
+                # write fail value and sets self.failFileID to its ID
```

Comment:
Also overly verbose. `# Write a boolean that determines wheter the job fails.` suffices.